user/managing-os/templates.md

```diff
@@ -62,6 +62,10 @@ In addition, these templates may be somewhat less stable, since the Qubes develo
  * [Arch Linux]
  * [CentOS]
  * [CentOS Minimal]
+ * [CentOS Xfce]
+ * [Gentoo]
+ * [Gentoo Minimal]
+ * [Gentoo Xfce]
```

Comment:
@fepitre, these links are missing, and I don't see any CentOS Xfce or Gentoo Xfce docs.

Reply:
Ah yes they need to be added. Forgot to add the pages sorry.

Reply:
No problem. I've added links for `[Gentoo]` and `[Gentoo Minimal]`, and I've removed the `[CentOS Xfce]` and `[Gentoo Xfce]` entries for now. Please feel free to add them back when those pages are ready.